doc/sphinx/changes.rst

```diff
@@ -18,7 +18,99 @@ Version 9.1
 Summary of changes
 ~~~~~~~~~~~~~~~~~~
 
-TODO
+We highlight some of the most impactful changes here:
+
+- fixed incorrect guard checking leading to inconsistencies (multiple PRs)
+
+- sort polymorphic universe instances should now be written
+  as `@{s ; u}` instead of `@{s | u}`
+  (`#20635 <https://github.com/rocq-prover/rocq/pull/20635>`_)
+
+- fixed handling of notation variables for ltac2 in notations
+  (i.e. `Notation "'foo' x" := ltac2:(...)`)
+  (`#20313 <https://github.com/rocq-prover/rocq/pull/20313>`_)
+
+- attribute :attr:`refine` support for :cmd:`Definition`
+  (`#20355 <https://github.com/rocq-prover/rocq/pull/20355>`_)
+
+- Rocq can be compile-time configured to be relocatable
+  (`#19901 <https://github.com/rocq-prover/rocq/pull/19901>`_)
+
+- extraction handles sort polymorphic definitions
+  (`#20655 <https://github.com/rocq-prover/rocq/pull/20655>`_)
+
+See the `Changes in 9.1.0`_ section below for the detailed list of changes,
+including potentially breaking changes marked with **Changed**.
+Rocq's `reference manual for 9.1 <https://rocq-prover.org/doc/v9.1/refman>`_,
+documentation of the 9.1 `core <https://rocq-prover.org/doc/v9.1/corelib>`_ and
+`standard <https://rocq-prover.org/doc/v9.1/stdlib>`_ libraries,
+`reference manual of the 9.1 standard library <https://rocq-prover.org/doc/v9.1/refman-stdlib>`_
```

Comment:
FWIW, there is no 9.1 Stdlib (it should probably be released at some point but if nobody does it, it just doesn't happen)